src/main/resources/standardTestCases/fofn_caching.test

```diff
@@ -0,0 +1,13 @@
+name: fofn_caching
+testFormat: runtwiceexpectingcallcaching
+backends: [Jes-Caching-No-Copy]
+
+files {
+  wdl: fofn_caching/fofn_caching.wdl
+}
+
+metadata {
+  workflowName: fofn_caching
+  status: Succeeded
+  "calls.fofn_caching.use_fofn.callCaching.result": "Cache Hit: <<CACHE_HIT_UUID>>:fofn_caching.use_fofn:-1"
```

Comment:
I think I must have missed something in the FoFN discussion; why is this expected to cache hit?